idemp-rest-test/testing/src/utils/Utils.ts

```diff
@@ -145,11 +149,13 @@ export async function createInvoice(valueObject: ValueObject) {
 		businessPartner: valueObject.businessPartner,
 		dateInvoiced: valueObject.date?.toISOString(),
 		invoiceLines: [],
+		isSalesOrderTransaction: valueObject.isSalesOrderTransaction,
 	};
 	const invoiceLine: Partial<InvoiceLine> = {
```

Comment:
From Chuck's video he mentions these methods are for what most people do most of the time. If we follow that, I feel most of the time we should set the product and not the charge. Then, whoever wants to do something differently can add something specific in the test (after this method is called) or not use the method. Do we want to go that route?

Reply:
Yeah, that makes sense. Thanks

Reply:
Coming to think of it, how do we gauge that a method is used by most people most of the time? Is it by the number of users using the functionality or by the number of features implementing it? 

For instance we have at least 3 functionalities i.e Service Debt (ARI), Track Expense (API) and Track Income (ARI) that use a charge against 1 i.e the visit page (ARI) which either uses a product and/or charge (payment). If this is true, shouldn't we then have a charge as the default and a product can be added whenever a test is required for it? Thoughts?

Reply:
You're right - this method is meant to be used after creating an order (see above on line 142), so it's only called after creating a sales or purchase order. For Debt Payments, Expenses, and Income, we'll probably need a new method. In that method, why don't we have it so that it can use charges by default (since we'll not invoice products that don't come from an order since that doesn't impact stock). If we ever do need invoices for products without an order, the person can just not use this method. How does that sound?

Reply:
cool.